### PR TITLE
added deviation lag type

### PR DESCRIPTION
**Pull request by Hobbydos**
added lag type when deviation is required

### CODE REVIEW

**Comment by Hobbydos** [comment]
@robshakir  @xw-g

File: feature/gnmi/ate_tests/telemetry_port_speed_test/telemetry_port_speed_test.go [hunk 1]
```diff
@@ -143,6 +143,7 @@ func (tc *testCase) setupAggregateAtomically(t *testing.T) {
 	}
 
 	agg := d.GetOrCreateInterface(tc.aggID)
+	agg.Type = ieee8023adLag
 	agg.GetOrCreateAggregation().LagType = tc.lagType
```
Comment (liulk) [inline feature/gnmi/ate_tests/telemetry_port_speed_test/telemetry_port_speed_test.go:147]: I was going to suggest fixing it elsewhere like here:

https://github.com/openconfig/featureprofiles/blob/d3d4ff7252906141715820ad0c48492f828c9a87/feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go#L161-L163

But looks like this is the only place we missed it. For the sake of consistency, let's reorder the lines similar to other places.

```suggestion
	agg := d.GetOrCreateInterface(tc.aggID)
	agg.GetOrCreateAggregation().LagType = tc.lagType
	agg.Type = ieee8023adLag
```

**Comment by xw-g** [comment]
LGTM. background is internal cl/445201506.

**Review by xw-g — APPROVED** [review]

**Comment by xw-g** [comment]
Merge pending @liulk's approval/LGTM.

**Review by liulk — APPROVED** [review]

**Pull request merged**